### DISCUSSION TITLE
SmartBear Supported

### DISCUSSION THREAD

**Discussion by mefellows** [pact-foundation/roadmap#12] [Ideas]
TBC

TL;DR - the implementation of the next evolution of https://docs.pact.io/help/smartbear

Item | Description | Link | Owner | Status
------|------------|------|-------|--------
Issue Triaging | Create an issue triage process that covers key Pact Foundation assets | https://github.com/pact-foundation/devrel/discussions/15 | @mefellows | 🟠 In progress 
Roadmaps | Improve the current roadmap process, including process and visibility, that encompasses the Pact estate | https://github.com/pact-foundation/devrel/discussions/14 | @mefellows | 🟠 In progress 
Define supported assets | Agree on the repositories and channels in scope for support. | [Assets](https://github.com/search?q=org%3Apact-foundation%20topic%3Asmartbear-supported&type=repositories) | @mefellows  | 🟢 Done
“Community Supported” | What does it mean for a project to be community supported? | link | @mefellows | TODO
“SmartBear Supported” | What does it mean for a project to be SmartBear supported? | link | @mefellows | TODO
Define SLA | Define the commitments we will provide to defined OSS assets | link | @mefellows  | TODO
